### PR TITLE
fix(tonemapping): ACT-956 tonemapping fix

### PR DESCRIPTION
**Pull request by Jarvv**
Changes:
- Renabled tone mapping on materials with an emissive map
- Fixed attachment meshes not being parented correctly on fallback

Old:
Screenshot 2024-09-10 at 16 04 57

New:
Screenshot 2024-09-10 at 16 04 20

### CODE REVIEW

File: src/services/Models.service.tsx [hunk 1]
```diff
@@ -287,12 +286,17 @@ function buildFallback(nodes: Nodes, transform: Transform = new Transform()): JS
         if (node.type === 'Mesh') {
           return (
             <mesh
+              attach={(parent, self) => {
+                parent.add(self);
+                self.parent = node.parent;
+                return () => parent.remove(self);
+              }}
```
Comment (Jarvv) [inline src/services/Models.service.tsx:293]: Add the mesh as a child to the <group>, but reparent the nodes object3D to the object3D of its parent node

**Comment by ericjansenrpm** [comment]
Looks great James! Please also test the lighting with wings and other attachments. :) For the updated tonemapping try how it looks like with the optional bloom postprocessing that we provide. You can find examples in the storybook.

File: src/components/Lights/Lights.component.tsx [hunk 2]
```diff
@@ -19,9 +19,9 @@ export const LIGHT_CONFIG = Object.freeze({
     fillLightIntensity: 3.0,
```
Comment (ericjansenrpm) [inline src/components/Lights/Lights.component.tsx:19]: let also try to reduce the orange light - we got some feedback there

**Review by ericjansenrpm — CHANGES_REQUESTED** [review]
Let's create the rc together manually after the storybook was updated. :)

**Review by ericjansenrpm — APPROVED** [review]

**Comment by Jarvv** [comment]
Tone-mapping fix validated inside local nova-editor

Screenshot 2024-09-13 at 11 58 39

**Pull request merged**